pgcontents/alembic/env.py

```diff
@@ -15,10 +15,10 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-from os.path import dirname, abspath
-import sys
```

Comment:
Do these have to be in the namespace for alembic to run properly?  If not they're probably just cruft.